locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1250 @@
+msgid "Orthophoto Extent"
+msgstr ""
+
+#: app/models/task.py:232
+msgid "DSM Extent"
+msgstr ""
+
+#: app/models/task.py:233
+msgid "DTM Extent"
+msgstr ""
+
+#: app/models/task.py:237
+msgid ""
+"A requested action to be performed on the task. The selected action will be "
+"performed by the worker at the next iteration."
+msgstr ""
+
+#: app/models/task.py:237
+msgid "Pending Action"
+msgstr ""
+
+#: app/models/task.py:239
+msgid "A flag indicating whether this task is available to the public"
+msgstr ""
+
+#: app/models/task.py:239
+msgid "Public"
+msgstr ""
+
+#: app/models/task.py:240
+msgid ""
+"When set to a value different than -1, indicates that the images for this "
+"task have been / will be resized to the size specified here before "
+"processing."
+msgstr ""
+
+#: app/models/task.py:240
+msgid "Resize To"
+msgstr ""
+
+#: app/models/task.py:243
+msgid ""
+"Value between 0 and 1 indicating the upload progress of this task's files to "
+"the processing node"
+msgstr ""
+
+#: app/models/task.py:244
+msgid "Upload Progress"
+msgstr ""
+
+#: app/models/task.py:247
+msgid ""
+"Value between 0 and 1 indicating the resize progress of this task's images"
+msgstr ""
+
+#: app/models/task.py:248
+msgid "Resize Progress"
+msgstr ""
+
+#: app/models/task.py:251
+msgid ""
+"Value between 0 and 1 indicating the running progress (estimated) of this "
+"task"
+msgstr ""
+
+#: app/models/task.py:252
+msgid "Running Progress"
+msgstr ""
+
+#: app/models/task.py:254
+msgid "URL this task is imported from (only for imported tasks)"
+msgstr ""
+
+#: app/models/task.py:254
+msgid "Import URL"
+msgstr ""
+
+#: app/models/task.py:255
+msgid "Number of images associated with this task"
+msgstr ""
+
+#: app/models/task.py:255
+msgid "Images Count"
+msgstr ""
+
+#: app/models/task.py:256
+msgid ""
+"A flag indicating whether this task is currently waiting for information or "
+"files to be uploaded before being considered for processing."
+msgstr ""
+
+#: app/models/task.py:256
+msgid "Partial"
+msgstr ""
+
+#: app/models/task.py:260
+msgid "Tasks"
+msgstr ""
+
+#: app/models/task.py:269
+msgid "unnamed"
+msgstr ""
+
+#: app/models/task.py:372
+msgid "Importing assets..."
+msgstr ""
+
+#: app/models/task.py:411 app/models/task.py:679
+msgid "Invalid zip file"
+msgstr ""
+
+#: app/models/task.py:505
+#, python-format
+msgid "Connection error: %(error)s"
+msgstr ""
+
+#: app/models/task.py:585
+msgid "Cannot restart a task that has no processing node"
+msgstr ""
+
+#: app/models/task.py:737
+#, python-format
+msgid "Unsupported SRS %(code)s. Please make sure you picked a supported SRS."
+msgstr ""
+
+#: app/models/task.py:747
+msgid "Done!"
+msgstr ""
+
+#: app/models/theme.py:16
+msgid "Name of theme"
+msgstr ""
+
+#: app/models/theme.py:19
+msgid "Most text, icons, and borders."
+msgstr ""
+
+#: app/models/theme.py:19
+msgid "Primary"
+msgstr ""
+
+#: app/models/theme.py:20
+msgid "The main background color, and text color of some buttons."
```

Comment:
```suggestion
msgid "The main background colour, and text colour of some buttons."
```

Reply:
`color` is mainstream and preferred in `en_US`.

Reply:
But think about how nice it would be if nothing was wrong in the world. :)